prisma-fmt-wasm/scripts/check.sh

```diff
@@ -20,12 +20,12 @@ echo 'ok.'
 
 echo '2. We can call the module directly and get back a valid result.'
 
-VERSION_FROM_MODULE=`$node -e "const prismaFmt = require('$prisma_fmt_wasm'); console.log(prismaFmt.version())"`
+VERSION_FROM_MODULE=`$node -e "const prismaFmt = require('$prisma_fmt_wasm'); console.log(prismaFmt.format('meow', '{}'))"`
```

Comment:
nitpick: we should rename that variable, it doesn't have anything to do with versions anymore

Reply:
At some point, we should also rename the module